services/socket.js

```diff
@@ -1,18 +1,24 @@
 const rollResult = require('../helper/rollResult');
 const calculateResult = require('../helper/calculateResult');
-const { TIME_LEFT_DEFAULT, PENDING_COUNTDOWN_TIME, COUNTDOWN_INTERVAL_SECOND } = require('../config');
+const { SECRET_KEY, TIME_LEFT_DEFAULT, PENDING_COUNTDOWN_TIME, COUNTDOWN_INTERVAL_SECOND } = require('../config');
+const jwt = require('jsonwebtoken');
 
 const activateSocket = (io) => {
   let timeLeft = TIME_LEFT_DEFAULT;
   let timeInterval;
   let userConnectedCounter = 0;
+  let userId;
 
-  const countdown = () => {
+  const countdown = async () => {
```

Comment:
Why you don't move ```countdown``` to another file than import to ```socker.js```, I think this function has separately logic